Config/config.php

```diff
@@ -3,7 +3,7 @@
 return [
     'name' => 'Custom Email Settings',
     'description' => 'Additional settings for emails to use different api key',
-    'version' => '2.0',
+    'version' => '2.01',
```

Comment:
Leave the version to 2.0, because for now we don't have any tag. This is bug fixing on staging, it's not a bug fix on production.

Reply:
this was done for testing purposes, i'll change it back in the final code